deploy/kuberhealthy-prometheus-operator.yaml

```diff
@@ -423,6 +423,9 @@ spec:
           requests:
             cpu: 400m
             memory: 300Mi
+          limits:
```

Comment:
These files with get re-generated based on the helm chart -- doing this will cause it to be overwritten.

Reply:
I can remove them here and in deploy/kuberhealthy-prometheus.yaml. Is this also the case for deploy/kuberhealthy.yaml and should those lines be removed as well?

Reply:
Yes, this is also the case! I think this is okay as it shouldn't conflict with the result of the re-generation